src/v/redpanda/application.cc

```diff
@@ -342,33 +342,33 @@ void application::hydrate_config(const po::variables_map& cfg) {
         }
     };
 
-        ss::smp::invoke_on_all([&config, cfg_path] {
-            config::node().load(cfg_path, config);
-        }).get0();
-
-        auto node_config_errors = config::node().load(config);
-        for (const auto& i : node_config_errors) {
-            vlog(
-              _log.warn,
-              "Node property '{}' validation error: {}",
-              i.first,
-              i.second);
-        }
-        if (node_config_errors.size() > 0) {
-            throw std::invalid_argument("Validation errors in node config");
-        }
+    ss::smp::invoke_on_all([&config, cfg_path] {
+        config::node().load(cfg_path, config);
+    }).get0();
 
-        // This includes loading from local bootstrap file or legacy
-        // config file on first-start or upgrade cases.
-        _config_preload = cluster::config_manager::preload(config).get0();
+    auto node_config_errors = config::node().load(config);
+    for (const auto& i : node_config_errors) {
+        vlog(
```

Comment:
> Whitespace update in subsequent commit, because all the indentation
jumps around & otherwise it's not obvious what I'm doing here.

wow you're not kidding